core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2551,6 +2624,10 @@ public void mount(AlluxioURI alluxioPath, AlluxioURI ufsPath, MountOptions optio
         throw e;
       }
       mMountTable.checkUnderWritableMountPoint(alluxioPath);
+      // Possible ufs sync.
+      syncMetadata(journalContext, inodePath, lockingScheme);
+      inodePath.validatePath(lockingScheme);
```

Comment:
does `mount` require the alluxio path to exist?

Reply:
mount actually doesn't want it to exist. That is checked in `mountAndJournal`.